pkgs/data/themes/qogir/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, gdk_pixbuf, librsvg, gtk-engine-murrine }:
+
+stdenv.mkDerivation rec {
+  name = "qogir-${version}";
```

Comment:
Why not?
```suggestion
  pname = "qogir-theme";
```

Reply:
Done.

Reply:
Question: Is `pname` and version automatically combined into the name with version? I've only ever seen `pname` in Python packages.

Reply:
Yes, with `buildGoPackage`, `stdenv.mkDerivation` and Python packages.